### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by mayhewsw**
Fixed documentation in preparation for the gh-pages branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +28,5 @@ From the root directory run the following command:
 ## Licensing
 To see the full license for this software, see [LICENSE](LICENSE) or visit the download 
 page for this software and press "Download". The next screen displays the license. 
+
+Cool!
```
Comment (danyaljj) [inline README.md]: Drop this?

File: lbjava/doc/LBJLANGUAGE.md [hunk 2]
```diff
@@ -683,55 +687,3 @@ line parameters (see Section 6.2). For example, if the classifiers in an LBJava
 to take classes from the programmer’s internal representation as input, the LBJava compiler will
 automatically compile the Java source files containing those class’ implementations if their class
 files don’t already exist or are out of date.
-
```
Comment (danyaljj) [inline lbjava/doc/LBJLANGUAGE.md]: Removed intentionally?

File: lbjava/doc/ALGORITHMS.md [hunk 3]
```diff
@@ -1,94 +0,0 @@
-# Learning Algorithms
```
Comment (danyaljj) [inline lbjava/doc/ALGORITHMS.md]: Removed intentionally?

Comment (mayhewsw) [inline lbjava/doc/ALGORITHMS.md]: Yes. There was no content.

Comment (danyaljj) [inline lbjava/doc/ALGORITHMS.md]: huh? What do you mean? I can see the content ...

Comment (danyaljj) [inline lbjava/doc/ALGORITHMS.md]: Or maybe you mean, content as in "tutorial"?

Comment (mayhewsw) [inline lbjava/doc/ALGORITHMS.md]: [Daniel proved his point via personal communication]

File: lbjava/doc/LBJLANGUAGE.md (hunk 2 above)
Comment (danyaljj) [inline lbjava/doc/LBJLANGUAGE.md]: Hb this?

**Comment by danyaljj** [comment]
Looks good. Merging!

**Pull request merged**